lambdas/build-cri-oauth-request/src/main/java/uk/gov/di/ipv/core/buildcrioauthrequest/BuildCriOauthRequestHandler.java

```diff
@@ -312,7 +312,7 @@ private JWEObject signEncryptJar(
 
         var vcs =
                 configService.enabled(SESSION_CREDENTIALS_TABLE_READS)
-                        ? sessionCredentialsService.getCredentials(
+                        ? sessionCredentialsService.getLatestCredentialsByIssuer(
```

Comment:
Presumably this is so we send the correct address in the shared claims?

I'm wondering if this is really the right approach - would it be better to move this logic to the bit that extracts the shared claims, i.e. distinguishing:
- 'We ignore any VCs that aren't the latest from a given issuer'
- 'When finding the address/name/dob, we use the latest VC with that information'

(I'm less convinced the first condition will be true forever than the latter!)

Reply:
Yeah, that's why this is here. Thought having the convenience method would be useful. It's used again in the build-user-identity-lambda for similar reasons.
I did actually consider making `getCredentials` do this and not adding the extra method, as I actually don't think it would be an issue for us to do this everywhere currently.
When do you forsee us needing to reference an older VC from an issuer from the users current session?